CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ The sections should follow the order `Added`, `Changed`, `Fixed`, and `Removed`.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+
+- On macOS, read `AppleFontSmoothing` from user defaults to decide whether fonts should be "smoothed".
+
+### Changed
+
+- `darwin::Rasterizer` renamed to `darwin::CoreTextRasterizer`
```

Comment:
```suggestion
- Renamed `darwin::Rasterizer` to `darwin::CoreTextRasterizer` on macOS
```

Reply:
"on macOS" is redundant here, no? It's renamed for everybody, after all, it's just a module that is only used on macOS.

Reply:
Moving `Renamed` to the front.